src/views/TradingCompetition/components/TeamRanks/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex, Box, Image } from '@pancakeswap-libs/uikit'
+import CakerBunny from '../../pngs/cakers.png'
+import TopTradersCard from './TopTradersCard'
+import Podium from './Podium'
+
+const Wrapper = styled(Flex)`
+  flex-direction: column;
+
+  ${({ theme }) => theme.mediaQueries.md} {
+    flex-direction: row;
+  }
+`
+
+const StyledPodiumWrapper = styled(Flex)`
+  margin: 0 0 40px 0;
+  ${({ theme }) => theme.mediaQueries.md} {
+    flex: 1;
+    margin: 0 40px 0 0;
+  }
+`
+
+const BunnyImageWrapper = styled(Box)`
+  display: none;
+
+  ${({ theme }) => theme.mediaQueries.md} {
+    display: flex;
+    width: 200px;
+    height: 205px;
+  }
+`
+
+const StyledTopTradersWrapper = styled(Flex)`
+  flex: 1;
+  ${({ theme }) => theme.mediaQueries.md} {
+    flex: 2;
+  }
+`
+
+const TeamRanks = () => {
+  return (
+    <>
```

Comment:
Does this need to be wrapped in a fragment?

Reply:
Ah perhaps once... not any more. Good spot